src/transformers/models/owlv2/configuration_owlv2.py

```diff
@@ -209,7 +209,7 @@ def __init__(
         num_attention_heads=12,
         num_channels=3,
         image_size=768,
-        patch_size=32,
+        patch_size=16,
```

Comment:
From @NielsRogge  (offline)

> Hi, yes that might be a mistake, since the V2 checkpoints don't have a patch size of 32 (only 16 for the base versions and 14 for the large ones)

Since this is recent + this is really not correct ==> update should be fine